packages/telegraf/windows.build.ps1

```diff
@@ -12,5 +12,6 @@ $Env:TMP="C:\tmp"
 Set-Location C:\gopath\src\github.com\influxdata\telegraf\
 $env:GOOS="windows"
 $env:GOARCH="386"
+$env:GOPATH="c:\gopath"

```

Comment:
```suggestion
```

Reply:
Why should this be unset?

Reply:
Looks like Windows build just passed

Reply:
I set this because I though it might solve the issue with windows build.